model/src/pyrenew/process/firstdifferencear.py

```diff
@@ -72,7 +72,11 @@ def sample(
             inits=jnp.atleast_1d(init_rate_of_change),
             name=name + "_rate_of_change",
         )
-        return (init_val + jnp.cumsum(rates_of_change.flatten()),)
+        return (
+            SampledValue(
+                init_val + jnp.cumsum(rates_of_change.value.flatten())
+            ),
+        )
```

Comment:
As above, I think we should probably pass `self.t_start` and `self.t_unit` here, even if they're typically going to be `None`.